tests/components/modbus/conftest.py

```diff
@@ -44,6 +45,7 @@ async def base_test(
     check_config_only=False,
     config_modbus=None,
     scan_interval=None,
+    mock_hook=None,
```

Comment:
The hock is something local to conftest.py, we do not want the single test files to care about hooks.